### ISSUE TITLE
nyroModal and jQuery 1.8.1

### ISSUE THREAD

**Issue by DarkoIgnjatovic** [nyroDev/nyroModal#195]
It seems that latest nyroModal isn't compatible with latest jQuery (1.8.1).

After upgrading jQuery,  images are not showing up in modal, I'm getting just nyro background.

In firebug I'm getting  an error:  <code>TypeError: $.curCSS is not a function</code>

**Comment by SailorsGrave** [comment]
I just ran into the same issue and fixed it by changing function _getCurCSS (around line 687) to this:

``` javascript
_getCurCSS: function(elm, name) {
    var ret = parseInt($(elm).css(name));
    return isNaN(ret) ? 0 : ret;
}
```

Problem ist that .curCSS became obsolete since jQuery 1.8 . Hope this helps.

**Comment by Zurd** [comment]
Thanks SailorsGrave, works just fine now! Wasn't easy to edit it in a .min file with nano in a command line though but not impossible ^.^

**Comment by dsnegi** [comment]
Well, it still not working with it can you guys please put the updated source files? Thanks =)

**Comment by nyroDev** [comment]
This should be fixed by commit dec12d57f40259bf996cc84faec632b8cb7a44ae
